docs/source/conf.py

```diff
@@ -15,22 +15,51 @@
 import os
 import sys
 
+from astropy.io.misc import yaml
+
+import simtools.version
+
 sys.path.insert(0, os.path.abspath("../../simtools"))
 sys.path.insert(0, os.path.abspath("../../applications"))
 sys.path.insert(0, os.path.abspath("../.."))
 
 
+def get_authors_from_citation_file(file_name):
+    """
+    Read list of authors from CITATION.cff file
+
+    """
+    try:
+        with open("../../CITATION.cff") as file:
+            citation = yaml.load(file)
+    except FileNotFoundError:
+        raise
+
+    author = ""
+    try:
+        for person in citation["authors"]:
+            author = author + person["given-names"] + " " + person["family-names"]
+            author += " (" + person["affiliation"] + "), "
+    except KeyError:
+        pass
+    return author[:-1]
```

Comment:
Wouldn't this remove the last author?

Reply:
No, this removes the last `,` from the string build in this loop.